src/components/ProductListContainer/styles.css

```diff
@@ -0,0 +1,5 @@
+.cards {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
```

Comment:
I think the styles files have too much indentation, two spaces is fine (the same as normal code)